modules/mbedtls/Kconfig.tls-generic

```diff
@@ -387,12 +387,22 @@ config MBEDTLS_HAVE_ASM
 config MBEDTLS_ENTROPY_C
 	bool "Mbed TLS entropy accumulator"
 	depends on MBEDTLS_SHA256 || MBEDTLS_SHA384 || MBEDTLS_SHA512
-	default y if MBEDTLS_ZEPHYR_ENTROPY
 	help
 	  This module gathers entropy data from enabled entropy sources. It's
 	  mostly used in conjunction with CTR_DRBG or HMAC_DRBG to create
 	  a deterministic random number generator.
 
+config MBEDTLS_ZEPHYR_ENTROPY_POLL
```

Comment:
Thinking about the naming of this... Would `MBEDTLS_POLL_ZEPHYR_ENTROPY` (or `MBEDTLS_ENTROPY_POLL_ZEPHYR` express more clearly that `Mbed TLS polls Zephyr for entropy`?